docs/user/system_upgrades.md

```diff
@@ -0,0 +1,92 @@
+# Upgrading your system
+
+Brewblox is designed to be compatible with the majority of Linux-based systems.
+but there still are dependencies, and those have their own End of Life dates.
+
+When your system is (soon to be) no longer supported, we strongly recommend upgrading.
+In the long run, it takes much more effort to keep an old system running than it is to upgrade.
+
+To assist with upgrades, `brewblox-ctl` can save and load snapshots of your installation.
+This lets you easily move your configuration between systems.
+
+## Deprecation schedule
+
+We track two deprecation schedules: Debian (the base of the Raspberry Pi OS), and Python.
+Linux distros come with a built-in version of Python.
+For some systems, it's doable to upgrade Python versions.
+For the Raspberry Pi, it's significantly easier and faster to upgrade the entire OS.
+
+### Raspberry Pi OS
+
+The Raspberry Pi OS is based on Debian,
+and we support the releases based on the [stable Debian version](https://wiki.debian.org/DebianReleases).
+Debian versions become stable a year after release.
```

Comment:
I would mainly note that we support versions that are not EOL, and not talk about what defines stable.